reconcile/cli.py

```diff
@@ -164,6 +165,14 @@ def run_integration(func, *args):
     except RunnerException as e:
         sys.stderr.write(str(e) + "\n")
         sys.exit(1)
+    except GqlApiError as e:
+        if '409' in str(e):
```

Comment:
a bit fragile, can we detect the exit code a bit more robustly?

Reply:
agreed to improve the GqlApiError in the future

Reply:
add a `TODO` and let's leave this for the future.